graphql_compiler/compiler/emit_sql.py

```diff
@@ -435,7 +533,8 @@ def backtrack(self, previous_location):
     def traverse(self, vertex_field, optional):
         """Execute a Traverse Block."""
         if self._current_fold is not None:
-            raise NotImplementedError('Traversals inside a fold are not implemented yet.')
+            raise NotImplementedError('Traversals inside a fold are not implemented '
+                                      'yet {}.'.format(self))
```

Comment:
`self` here is a `CompilationState`, right? I don't see a `__str__` method for it so how does it look when printed? I don't know if this is necessary information to give the user.